CryptoBook/api.py

```diff
@@ -48,4 +48,4 @@ def load_config():
 # 'pragma' line below insures Coverall does not bother checking this function for coverage.
 if __name__ == '__main__': # pragma: no cover
     host, port = load_config()
-    app.run(host=config['host'], port=config['port'])
+    app.run(host=host, port=port)
```

Comment:
This was a small bug that escaped me, and since all tests ran with success was pushed into master by mistake by me.

Reply:
Fixed with #16 -- for review for other folks.